src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const threeDaysDiscount = 20;
+  const SevenDaysDiscount = 50;
```

Comment:
don't create constant in `camel case` from `uppercase` letter

Reply:
sorry, made a typo. now fixed. thanks!